release/release_tests.yaml

```diff
@@ -4208,7 +4208,7 @@
   team: data
   cluster:
     cluster_env: app_config.yaml
-    cluster_compute: aggregate_benchmark_compute.yaml
+    cluster_compute: single_node_benchmark_compute.yaml
```

Comment:
Make it specific to read_benchmark since we may have single node yaml for other APIs.

Reply:
I was intentionally share the same yaml file across single node benchmarks. we want to have a specific single node yaml file for each API?

Reply:
It might be different (not necessarily for each API) as different machine type have different cpu/memory etc. But this one seems fine to keep it for now until we do want to diverge the setup.